x-pack/package.json

```diff
@@ -170,6 +172,7 @@
     "humps": "2.0.1",
     "icalendar": "0.7.1",
     "inline-style": "^2.0.0",
+    "iron": "4",
```

Comment:
Let's make sure @joshdover is aware of this, so that it lines up with the version we are requesting in the hapi upgrade (just in case it changed, I'm not sure if it has or not)

Reply:
This will likely need to be bumped with the hapi upgrade, even though the newer version is "wire compatible" and should serialize/deserialize the same.